src/contract_core/contract_def.h

```diff
@@ -137,7 +138,7 @@ contractSystemProcedures[contractIndex][END_EPOCH] = (SYSTEM_PROCEDURE)contractN
 contractSystemProcedures[contractIndex][BEGIN_TICK] = (SYSTEM_PROCEDURE)contractName::__beginTick;\
 contractSystemProcedures[contractIndex][END_TICK] = (SYSTEM_PROCEDURE)contractName::__endTick;\
 contractExpandProcedures[contractIndex] = (EXPAND_PROCEDURE)contractName::__expand;\
-_##contractName->__registerUserFunctionsAndProcedures(qpi);
+    _##contractName->__registerUserFunctionsAndProcedures(qpi);

```

Comment:
Please undo this change and do not change code outside the scope of your SC and SC init. It is blowing up the diffs and makes tracking of changes more difficult.